### ISSUE TITLE
TLS1.3 support

### ISSUE THREAD

**Issue by jpds** [coturn/coturn#630]
Any plans for adding TLS1.3 support?

**Comment by misi** [comment]
@jpds Unfortunately I don't have time to implement it in foreseeable future, but I would very happy to merge a Pull Request with it.
It is a needed feature, but unfortunately I don't have capacity and enough knowledge to implement it. Any help would be great!

**Comment by jpds** [comment]
I was debugging a coturn server I run today, when I noticed that the `openssl` client I was using was talking TLS1.3 to it?

```
$ openssl s_client -connect turn.test:5349
...
---
SSL handshake has read 1384 bytes and written 391 bytes
Verification: OK
---
New, TLSv1.3, Cipher is TLS_AES_256_GCM_SHA384
Server public key is 256 bit
Secure Renegotiation IS NOT supported
Compression: NONE
Expansion: NONE
No ALPN negotiated
Early data was not sent
Verify return code: 0 (ok)
---
---
Post-Handshake New Session Ticket arrived:
SSL-Session:
    Protocol  : TLSv1.3
    Cipher    : TLS_AES_256_GCM_SHA384
...
```

coturn version is `4.5.1.1-1.1+deb10u1`.

**Comment by SirCypher** [comment]
Coturn itself might not support TLSv1.3 right now, but the OpenSSL libraries of most OS do.
That's when you get this strange mismatch where the application does not officially support 1.3, but you can connect anyway.

The downside is that you can't disable TLSv1.3 or modify the cipher list from within the coturn configuration file. All those values will only affect TLSv1.2 and below.
If you want to modify those things for 1.3 you need to do so in the global openssl configuration, usually at "/etc/ssl/openssl.cnf".

I'm running Debian with latest coturn and a few TLSv1.3 clients and so far I didn't encounter any errors.

**Comment by eakraly** [comment]
Addressed in #989
When building with openssl version that supports TLS1.3 you automatically get support
